buttons-examples/iOS/ios-button.md

```diff
@@ -480,31 +370,44 @@ Icons can be used as toggle buttons when they allow selection, or deselection, o
 
 The iOS icon toggle button is only available for use with the iOS [card](../Cards) component. Go to the card article for an [example](../Cards/#card-example-with-icon-buttons).
 
-<!--Theming for Shrine is not available as of 2020.02.26
 ## Theming buttons
 
-Buttons support [Material Theming](https://material.io/components/buttons/#theming) and can be customized in terms of color, typography and shape.
+You can theme an MDCButton to match any of the Material Button styles using theming
+extensions. [Learn more about theming extensions](../../docs/theming.md).
+
+### Buttons theming example
 
-### Button theming example
+To make use of the theming methods shown in the examples above do the following:
 
-API and source code:
+1. Install the theming extensions with Cocoapods
+    Add the following line to your `Podfile`:
 
-* `MaterialButton` (a subclass of [UIButton](https://developer.apple.com/documentation/uikit/uibutton))
-    * [Class description](https://)
-    * [GitHub source](https://github.com/material-components/)
+    ```
+    pod MaterialComponents/Buttons+Theming
+    ```
     
-The following example shows text, outlined and contained button types with Material Theming.
+    Run the installer:
+    
+    ```
+    pod install
+    ```
 
-!["Button theming examples for iOS with pink and black buttons and cut corners."](assets/button-theming.svg)
```

Comment:
Are there no screenshots?

Reply:
Should I put examples of all the types of buttons in the theming section? Since theming is the mechanism used to achieve the various button subtypes? It may seem like duplication to have screenshots down here in addition to the images in the Buttons subtype sections above.

Reply:
Yeah, but it's to show how Shrine theming would apply because developers might find it handy. So can you have a block of buttons like the Android buttons doc?

https://github.com/mingjane-work/doc-material-components/blob/mingjane-doc-branch/buttons-examples/Android/android-button-example.md

Reply:
I added a screenshot of buttons with Shrine theming in 166a821. The outlines on outlined buttons are supposed to be the on surface color at 12% opacity. The on surface color for Shrine is supposed to be 0xFFFBFA, which is a very light color. At 12% and 1 point (iOS rough equivalent of dp) thickness it's not even visible, at least to me. With that in mind, is it worth having there?

Reply:
Omg I checked again and was wrong! 0x442B2D is the on surface. I fixed it and fixed the screenshot. Sorry about that.